deploy/helm-chart/templates/deployment-promscale.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     chart: {{ template "promscale.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: "promscale-connector"
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/part-of: "promscale-connector"
+    app.kubernetes.io/component: "connector"
```

Comment:
I don't understand why this component is called `connector`, but everything else is called `promscale-connector`.

Reply:
Value of `app.kubernetes.io/component` should be saying what it the component role within the architecture. In case of our architecture `promscale` is a `connector` or `receiver` or `forwarder`, or something else that we want to call it (name is arbitrary). I did not use `promscale-connector` as prefix `promscale-` is not usually used to define anything within deployment architectures.

Reply:
Thanks for the explanation. I think I'm a little confused by whether `app.kubernetes.io/component` refers to a component name, or component type (having read what I've written, the distinction seems a little arbitrary, but I guess it could be answered by "Is it _a_ connector in any architecture (component type), or is it _the_ connector in this architecture (component name)?"). The docs that I've found don't seem to be super clear on that. Anyway,  I don't think that it _really_ matters.

Reply:
Commonly it is `type`. `name` is referenced by `app.kubernetes.io/name`.